src/main/java/racingcar/RacingCar.java

```diff
@@ -0,0 +1,16 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCar {
+    public String name;
+    public int position;
+    public List<Integer> randomNumbers;
```

Comment:
캡슐화와 불변 객체에 대해서 알아봅시다

```suggestion
public class RacingCar {
    private final String name;
    private final int position;
    private final List<Integer> randomNumbers;
```